app/uk/gov/hmrc/trackingconsentfrontend/assets/src/common/pageHandler.ts

```diff
@@ -0,0 +1,10 @@
+import enableGtm from "../interfaces/gtm";
+import {UserPreferences} from "../../types/UserPreferences";
+
+const pageHandler = (userPreference: UserPreferences, pageRenderer) => {
```

Comment:
I think this should have some tests around it.

Reply:
There are tests for this in tests/common/pageHandler.spec.ts